app/assets/javascripts/oxalis/controller/url_manager.js

```diff
@@ -65,7 +65,8 @@ class UrlManager {
     if (stateString) {
       const stateArray = stateString.split(",").map(Number);
       if (stateArray.length >= 5) {
-        state.position = Utils.numberArrayToVector3(stateArray.slice(0, 3));
+        const positionValues = stateArray.slice(0, 3).map(value => (!isNaN(value) ? value : 0));
```

Comment:
I would suggest to move the .map with the isNaN check two lines up so it is executed for all the numbers in the stateArray :) Currently, setting, for example, the zoomstep to a string value in the url hash results in an error as well.